apps/test/integration/levelTests.js

```diff
@@ -10,8 +10,8 @@
 // feedback when a user gets the puzzle wrong?
 
 import {assert} from '../util/configuredChai';
-import $ from 'jquery';
 import sinon from 'sinon';
+let $ = window.$ = window.jQuery = require('jquery');
```

Comment:
I think at the time I was thinking about how jQuery gets extended by plugins, so it's not really a `const `thing.  I guess it would only be for this scope though, which would make more sense.